backend/src/federation/validateCommunities.ts

```diff
@@ -0,0 +1,70 @@
+import { Community as DbCommunity } from '@entity/Community'
+import { IsNull } from '@dbTools/typeorm'
+// eslint-disable-next-line camelcase
+import { requestGetPublicKey as v1_0_requestGetPublicKey } from './client/1_0/FederationClient'
+// eslint-disable-next-line camelcase
+import { requestGetPublicKey as v1_1_requestGetPublicKey } from './client/1_1/FederationClient'
+import { backendLogger as logger } from '@/server/logger'
+import { ApiVersionType } from './enum/apiVersionType'
+
+export async function startValidateCommunities(timerInterval: number): Promise<void> {
+  logger.info(
+    `Federation: startValidateCommunities loop with an interval of ${timerInterval} ms...`,
+  )
+  while (true) {
```

Comment:
Very scary ...

Reply:
I suggest to validate the community if there is:
 * A difference over a certain time of `verfied_at`
 * and an outgoing request to the no longer verified community

Reply:
As we discussed, the while-loop is the simpliest solution, but in future there should be a trigger to validate a community announcement as soon the dht inserted or updated a community-entry.
But currently we only have a data-exchange over the database, but no control-flow exchange between our gradido modules (backend / dht / federation) like a trigger or eventing to validate only if necessary.